api/api.go

```diff
@@ -314,6 +315,7 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		apiAlert := &APIAlert{
 			Alert:  &a.Alert,
 			Status: status,
+			ID:     fmt.Sprintf("%d", a.Fingerprint()),
```

Comment:
No, it's just the `base 16, with lower-case letters for a-f`. See https://golang.org/pkg/fmt/#hdr-Printing